packages/plugin-hardhat/package.json

```diff
@@ -38,7 +38,7 @@
   "dependencies": {
     "@openzeppelin/defender-admin-client": "^1.48.0",
     "@openzeppelin/defender-base-client": "^1.48.0",
```

Comment:
Are we still actively using these? I see even proposals are included in the new [SDK](https://www.api-docs.defender.openzeppelin.com/#defender-sdk-proposal) but don't know about the status what are the Defender libraries we kept.